tabs/osd.js

```diff
@@ -774,7 +855,12 @@ OSD.msp = {
       // watch out, order matters! match the firmware
       result.push8(OSD.data.alarms.rssi.value);
       result.push16(OSD.data.alarms.cap.value);
-      result.push16(OSD.data.alarms.time.value);
+      if (semver.lt(CONFIG.apiVersion, "1.36.0")) {
```

Comment:
I still don't like that we have got bits of the MSP client implementation in this file, this makes maintenance very prone to errors.
All of this should be moved into `MSPHelper.js`, before more stuff is added here.